### ISSUE TITLE
[4.0] Menu 3rd level accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#25855]
This only happens with the final level of the menu tree expanded
![image](https://user-images.githubusercontent.com/1296369/62151287-e5962780-b2f7-11e9-8fee-dd1410f1e647.png)

I havent been able to find the problem yet - posting it here in case someone else wants to have a look
#### Issue
`<ul> and <ol> must only directly contain <li>, <script> or <template> elements` ([list](https://dequeuniversity.com/rules/axe/3.2/list?application=msftAI))
#### Target application
[Banners - extra files - Administration](http://localhost/back/administrator/index.php?option=com_banners&view=banners)
#### Element path
#menu-7
#### Snippet
`<ul id="menu-7" class="collapse-level-2 collapse in" style="">`
#### How to fix
    Fix all of the following:
      List element has direct children that are not allowed inside <li> elements
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by brianteeman** [comment]
I am assuming that this is related so posting it here

Looks like it needs quite a bit of work as NVDA is also having problems with the menu when expanded and announces `menu item` multiple times on expanded sub menu

```
Multilingual Associations
visited  link
menu item
menu item
News Feeds
visited  link
menu item
menu item
Smart Search
visited  link
menu item
menu item
Tags
visited  link
menu item
menu item
Users
collapsed  link
menu item
expanded
Dashboard: Users
visited  link
menu item
menu  subMenu
Manage
visited  link
menu item
menu item
Add New User
visited  link
menu item
menu item
Groups
visited  link
menu item
menu item
Access Levels
link
menu item
menu item

```

**Comment by brianteeman** [comment]
Found the problems. All caused because of a misunderstanding about building accessible menus. I am working on a fix but it might take a few days.

**Comment by dgrammatiko** [comment]
FWIW the menu should work with the keys up, down, left, right, **NOT** with tabs. Accessibility is totally ruined here...

**Comment by brianteeman** [comment]
It is not a menu - it is a set of navigation links

**Comment by dgrammatiko** [comment]
@brianteeman I must be drunk then: 
<img width="829" alt="Screenshot 2019-08-16 at 19 20 51" src="https://user-images.githubusercontent.com/3889375/63185708-05994b00-c05b-11e9-8673-4851c49ff7f0.png">

**Comment by brianteeman** [comment]
No you're just not understanding the issue.

**Comment by brianteeman** [comment]
Closed by #25909
